### ISSUE TITLE
Submit button when entering instance URL is not functional

### ISSUE THREAD

**Issue by David-Lor** [iprodanovbg/homeassistant-desktop#26]
I'm trying the app v1.5.3 running the AppImage from the Linux x86_64; my HomeAssistant instance is version 2023.3.1. Right-clicked the icon on the toolbar, clicked "Add another instance" or "Show window", and on the window where the HA instance URL is entered, the Submit button is greyed out, no matter which URL I enter (even the example one). Pressing Enter doesn't work.

![image](https://user-images.githubusercontent.com/17401854/233849044-82d3260a-934a-46cc-a053-bd5831056d81.png)

Logs when executing the appimage from terminal:

```bash
(base) david@ThinkpadE14-Debian:~$ ./Home-Assistant-Desktop-v1.5.3-linux-x86_64.AppImage 
17:32:12.795 › homeassistant-desktop started
17:32:12.810 › Platform: linux x64
17:32:13.109 › Checking for update
[110657:0423/173213.761480:ERROR:nss_util.cc(349)] After loading Root Certs, loaded==false: NSS error code: -8018
17:32:14.532 › Update for version 1.5.3 is not available (latest version: 1.5.3, downgrade is disallowed).
17:32:14.535 › Initialized main window
17:32:15.160 › Initialized Tray menu
17:32:15.189 › Initialized availability check
[110657:0423/173215.202821:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[110657:0423/173215.202946:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[110657:0423/173215.398282:ERROR:browser_main_loop.cc(269)] Gdk: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
```

EDIT: Found a similar issue (https://github.com/iprodanovbg/homeassistant-desktop/issues/19) but in my case the auth provider seems to be the correct one:

```curl
(base) david@ThinkpadE14-Debian:~$ curl --insecure https://homeassistant.pi.lan/auth/providers
[{"name":"Home Assistant Local","id":null,"type":"homeassistant"}]
```

Maybe it has something to do with the selfsigned certificate I'm using? On the example picture above I've tried typing the example URL, but my HA instance is actually running in the address shown in the curl command above, served behind an nginx reverse proxy with selfsigned certs, not added to the OS. Seeing the #19 issue I'm now assuming the app tries to connect to the server while typing, and the Submit button is only enabled once the connection with the instance can be successfully established?

**Comment by muecke36** [comment]
Same problem here

**Comment by muecke36** [comment]
The problem in my case is, that I enabled login without password (allow_bypass_login: true) [see here](https://www.home-assistant.io/docs/authentication/providers/#skip-login-page-examples) and therefore the login URL is not "/auth/providers" but "/auth/authorize".

**Comment by muecke36** [comment]
In version 1.4.11 the login with these settings works fine.

**Comment by muecke36** [comment]
Yeah that was the problem. I added the default auth provider and now also 1.5.3 is working for me:

auth_providers:
    - type: homeassistant

**Comment by chrisbati** [comment]
Hello, 

I have the same issue, "Submit" bouton is greyed out and I cannot click (on windows 10).
I add auth. providers, I restart but still the same.

When I check the auth providers I get:

> [{"name":"Home Assistant Local","id":null,"type":"homeassistant"}]

Thanks in advance for any help !

Chrisbati

My YAML config:

`# Loads default set of integrations. Do not remove.
default_config:

homeassistant:
  auth_providers:
    - type: homeassistant

'# Load frontend themes from the themes folder
frontend:
  themes: !include_dir_merge_named themes

http:
  ssl_certificate: /ssl/fullchain.pem
  ssl_key: /ssl/privkey.pem

automation: !include automations.yaml
script: !include scripts.yaml
scene: !include scenes.yaml

media_player:
  - platform: webrtc
    name: Dummy camera
    stream: some_stream
    audio: pcmu/48000
'

**Comment by Bashir-Rabbit** [comment]
Hi All,
i have identified issue, this URL login does not work with SSL_certificate/SSL_key if you remove it will work again. 
I do not want to use open text, but would love to use this app, is there any workaround to make SSL works ?
